packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFScriptMessageHandlerHostApi.m

```diff
@@ -12,7 +12,7 @@ - (void)userContentController:(nonnull WKUserContentController *)userContentCont
 @end
 
 @interface FWFScriptMessageHandlerHostApiImpl ()
-@property(nonatomic) FWFInstanceManager *instanceManager;
+@property(nonatomic, weak) FWFInstanceManager *instanceManager;
```

Comment:
Maybe a note in the declaration comment of `FWFInstanceManager` on expected ownership would be useful? I assume that this is to avoid retain loops, but it may be non-obvious that references to it should generally be weak (and who is responsible for having strong ownership of it.)